src/api.ts

```diff
@@ -15,14 +15,14 @@
  */
 
 export { Accessibility } from './accessibility';
-export { Browser } from './browser';
-export { BrowserContext } from './browserContext';
+export { BrowserBase as Browser } from './browser';
```

Comment:
These don't look like improvements either

Reply:
These fix bugs in apicoverage.spec.js. Previously we were exporting a type, which got stripped by TypeScript.